pulseapi/profiles/views.py

```diff
@@ -185,29 +194,34 @@ def qs(self):
         a legal filtering argument, we return an empty
         queryset, rather than every profile in existence.
         """
-        empty_set = UserProfile.objects.none()
+        qs = UserProfile.objects.none()
 
         request = self.request
         if request is None:
-            return empty_set
+            return qs
 
         queries = self.request.query_params
         if queries is None:
-            return empty_set
+            return qs
 
         fields = ProfileCustomFilter.get_fields()
         for key in fields:
             if key in queries:
-                return super(ProfileCustomFilter, self).qs
+                qs = super(ProfileCustomFilter, self).qs
+
+        if 'limit' in queries:
```

Comment:
can you make this a filter property? So it would be:

```python
limit = django_filters.NumberFilter(method='filter_limit')

def filter_limit(self, queryset, name, value):
    return queryset.limit(value)
```